src/common/containers/App.tsx

```diff
@@ -170,6 +171,7 @@ class App extends React.Component<AppProps, AppState> {
         <ErrorBoundary>
           <Switch>
             <Route exact path="/" component={Home} />
+            <Route path="//:dialog" component={Home} />
```

Comment:
Why two slashes?

Reply:
It's because `RouterDialog`s use double-slashes in the URL.